cfgov/unprocessed/js/modules/util/validators.js

```diff
@@ -41,9 +41,9 @@ function date( field, currentStatus ) {
 function email( field, currentStatus ) {
   var status = currentStatus || {};
   var regex =
-    '^[a-z0-9\u007F-\uffff!#$%&\'*+\/=?^_`{|}~-]+(?:\.[a-z0-9' +
+    '^[a-z0-9\u007F-\uffff!#$%&\'*+\/=?^_`{|}~-]+(?:\\.[a-z0-9' +
```

Comment:
Is this email validator still needed since you're validating it in `EmailSignup.js`?